src/librawspeed/decoders/Cr2Decoder.h

```diff
@@ -44,6 +44,13 @@ class Cr2Decoder final :
   RawImage decodeNewFormat();
   void sRawInterpolate();
   int getHue();
+
+  using yuv2rgb = void(int Y, int Cb, int Cr, int& r, int& g, int& b,
+                       const int* sraw_coeffs);
+  static yuv2rgb YUV_TO_RGB_v0;
+  static yuv2rgb YUV_TO_RGB_v1;
+  static yuv2rgb YUV_TO_RGB_v2;
```

Comment:
I see. I did not check disassebly indeed. I was trying to avoid making them a static functions, but if that is what it currently takes, ok.